tests/integration/enum-backed-with-cases-warning/input/index.rst

```diff
@@ -0,0 +1,30 @@
+===================
+PHP Enum with Const
```

Comment:
Heading does not match purpose (like given in the folder name)

Reply:
renamed